reviews/graduation-jaeger.md

```diff
@@ -0,0 +1,156 @@
+
+Jaeger project contains a number of sub-projects, such as client libraries, integrations with Kubernetes, etc. We do not grant committer rights equally across projects, only as needed. For example:
+
+  * Jaeger backend: Currently, we have 7 [committers][committers] and a few people contributing regularly. Full committers have commit rights across all other repositories.
+    * @black-adder (Uber)
+    * @jpkrohling (Red Hat)
+    * @objectiser (Red Hat)
+    * @pavolloffay (Red Hat)
+    * @tiffon (Uber)
+    * @vprithvi (Uber)
+    * @yurishkuro (Uber)
+  * Jaeger UI
+    * @tiffon (Uber)
+    * @saminzadeh (Uber)
+  * Jaeger client for C++
+    * @isaachier (Uber)
+  * Jaeger client for C#
+    * @Falco20019
+    * @grounded042 (Chatham Financial)
+    * @kevindaviscfc (Chatham Financial)
+    * @PhillipChaffee (Chatham Financial)
+    * @cwe1ss (c3-ls.com)
+
+We are on the constant lookout for new maintainers to join our ranks. In the past year over 200 people created commits and pull requests to Jaeger code base.
+
+## Demonstrate a substantial ongoing flow of commits and merged contributions.
+
+https://jaeger.devstats.cncf.io/d/4/companies-stats?orgId=1&var-period=m&var-metric=activity&var-repogroup_name=All&var-companies=All 
+
+## Have committers from at least two organizations.
+
+We currently have 7 committers from 2 organizations, Uber and Red Hat.
+
+## Have achieved and maintained a Core Infrastructure Initiative Best Practices Badge.
+
+https://bestpractices.coreinfrastructure.org/en/projects/1273 
+
+## Have completed an independent and third party security audit
+
+IN-PROGRESS: audit by Cure-53
+
+## Adopt the CNCF Code of Conduct.
+
+https://github.com/jaegertracing/jaeger/blob/master/CODE_OF_CONDUCT.md
+
+## Explicitly define a project governance and committer process.
+
+  * Governance: https://github.com/jaegertracing/jaeger/blob/master/GOVERNANCE.md
+  * Committers: https://github.com/jaegertracing/jaeger/blob/master/CODEOWNERS 
+
+## Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
+
+https://github.com/jaegertracing/jaeger/blob/master/ADOPTERS.md
+
+## Open-source alternatives to Jaeger
+
+### Apache Zipkin
+
+  * https://zipkin.apache.org/
+  * Incubating at Apache Software Foundation.
+  * Offers similar functionality to Jaeger, but lacks certain features, such as adaptive sampling or trace comparisons in the UI.
+  * Implemented in Java. Comes with its own instrumentation SDK for Java called Brave.
+
+### Apache Sky-Walking
+
+  * https://skywalking.apache.org/
+  * Top level project at Apache Software Foundation.
+  * An APM solution that combines tracing, metrics, and alerting.
+
+### Haystack
+
+  * https://expediadotcom.github.io/haystack/
+  * Tracing and analysis system from Expedia.
+  * Beyond simple tracing, allows to observe trends and setup anomaly detection.
+
+## Jaeger relationship to OpenTracing and OpenCensus
```

Comment:
Update this section to discuss Open Telemetry.

Reply:
done